performance_test/CMakeLists.txt

```diff
@@ -35,6 +35,28 @@ if(PERFORMANCE_TEST_FASTRTPS_ENABLED)
   add_definitions(-DPERFORMANCE_TEST_FASTRTPS_ENABLED)
 endif()
 
+# Use ODB support to create SQL database
+option(ODB_FOR_SQL_ENABLED "Enable ODB for SQL" OFF)
+
+if(ODB_FOR_SQL_ENABLED)
+  add_definitions(-DODB_FOR_SQL_ENABLED)
+  option(PERFORMANCE_TEST_ODB_SQLITE "Select DB type" ON)
```

Comment:
Is the default one `sql_lite` and `my_sql` will be switched on when required?

Reply:
Yes, it's written in the readme file - default is sqlite, but well, we can also make mysql default. I made the default sqlite because you don't need to have mysql server running and you just run ros2 run without any additional arguments, so it's easier.

Reply:
Okay great sounds good!